docs/AQL/master03-syntax.adoc

```diff
@@ -542,6 +568,7 @@ Class expressions syntax include three parts. A class expression must have part
 . part two (optional): AQL variable name
 . part three (optional): a standard predicate or an archetype predicate.
 
+[#_containment]
```

Comment:
This anchor is not needed - it will be generated automatically.

Reply:
ok, I had a reason here, ut I cannot recall now - I will remove this and see if it breaks something or not